File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.cc
#include <iostream>
using namespace  std;

int gcd(int a,int b){
	if(a<b){
		return gcd(b,a);
	}
	if(b==0){
		return a;
	}
	return gcd(b,a%b);
	
}

int main(){
	int a,b;
	cin>>a>>b;
	int GCD=gcd(a,b);
	int LCM=a*b/(GCD);
	cout<<GCD<<endl<<LCM;
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function __gcd(x, y) {
  if (x < y) [x, y] = [y, x];
  if (y == 0) return x;
  else return __gcd(y, x % y);
}

function solution(n, m) {
  let gcd = __gcd(n, m);
  let lcm = (n * m) / gcd;
  return [gcd, lcm];
}
